morango/models.py

```diff
@@ -14,12 +15,25 @@
 from django.utils import timezone
 from django.utils.six import iteritems
 from morango.utils.register_models import _profile_models
+from .utils import proquint
 
 from .certificates import Certificate, ScopeDefinition, Nonce, Filter
 from .manager import SyncableModelManager
 from .utils.uuids import UUIDField, UUIDModelMixin, sha2_uuid
 
 
+class MorangoInfo(models.Model):
```

Comment:
I don't think we need a model for this. It should be simpler and cleaner (and still fast, as this stuff is cheap) to use a base `ViewSet`, define the `retrieve` method, and just compute/return the data directly.

For the proquint, perhaps that logic can be embedded in a `get_proquint` method on the `InstanceIDModel`, and called from the viewset.